examples/mysql-kopia-repository-server/README.md

```diff
@@ -0,0 +1,438 @@
+
+### Create [Kopia Repository](https://kopia.io/docs/repositories/) using S3 as the location storage
+
+```bash
+$ kopia --log-level=error --config-file=/tmp/kopia-repository.config \
+--log-dir=/tmp/kopia-cache repository create --no-check-for-updates \
+--cache-directory=/tmp/cache.dir --content-cache-size-mb=0 \
+--metadata-cache-size-mb=500 --override-hostname=mysql.app \
+--override-username=kanisterAdmin s3 --bucket=<s3_bucket_name> \
+--prefix=/repo-controller/ --region=<s3_bucket_region> \
+--access-key=<aws_access_key> --secret-access-key=<aws_secret_access_key> --password=<repository_password>
+```
+
+### Create Repository Server CR
+
+**NOTE:**
+
+All the secrets mentioned below are required to be created in the same namespace as the Kanister controller.
+And all the secrets would be used while creating the Repository Server CR. 
+
+- Generate TLS Certificates and create TLS secret for Kopia Repository Server for secure communication between Kopia Repository Server and Client
+
+```bash
+$ openssl req -newkey rsa:2048 -nodes -keyout key.pem -x509 -days 365 -out certificate.pem
+
+$ kubectl create secret tls repository-server-tls-cert --cert=certificate.pem --key=key.pem -n kanister
+```
+
+- Create Location Secrets for Kopia Repository
+
+```bash
+# The following file s3_location_creds.yaml is a sample file for creating s3 credentials secrets. It contains the credentials for accessing the s3 bucket.
+$ vi s3_location_creds.yaml
+
+apiVersion: v1
+kind: Secret
+metadata:
+   name: s3-creds
+   namespace: kanister
+   labels:
+      repo.kanister.io/target-namespace: monitoring
+type: secrets.kanister.io/aws
+data:
+   # required: base64 encoded value for key with proper permissions for the bucket
+   aws_access_key_id: <base64_encoded_aws_access_key>
+   # required: base64 encoded value for the secret corresponding to the key above
+   aws_secret_access_key: <base64_encoded_aws_secret_access_key>
+```
+
+```
+$ kubectl create -f s3_location_creds.yaml -n kanister
+```
+
+```bash
+# The following file s3_location.yaml is a sample file for creating s3 location secrets. It contains the details of the s3 bucket.
+$ vi s3_location.yaml
+
+apiVersion: v1
+kind: Secret
+metadata:
+   name: s3-location
+   namespace: kanister
+   labels:
+      repo.kanister.io/target-namespace: monitoring
+type: Opaque
+data:
+   # required: specify the type of the store
+   # supported values are s3, gcs, azure, and file-store
+   type: czM=
+   bucket: <base64_encoded_s3_bucket_name>
+   # optional: used as a sub path in the bucket for all backups
+   path: <base_64_encoded_prefix_provided_when_creating_kopia_repository>
+   # required, if supported by the provider
+   region: <base64_encoded_s3_bucket_region>
+   # optional: if set to true, do not verify SSL cert.
+   # Default, when omitted, is false
+   #skipSSLVerify: false
+   # required: if type is `file-store`
+   # optional, otherwise
+   #claimName: store-pvc
```

Comment:
Remove these commented fields altogether since these are not required in this example